common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -10,13 +10,18 @@
 
 #include <bcl/beerocks_utils.h>
 #include <bcl/network/network_utils.h>
+#include <bcl/son/son_wireless_utils.h>
 
 #include <easylogging++.h>
+#include <net/if.h>
 
 #include <cmath>
+#include <functional>
 
 extern "C" {
 #include <dwpal.h>
+#include <slibc/stdio.h>
```

Comment:
This won't work with UGW - this is exatcly why we have the `HAVE_SAFEC` flags.
You need (here and in all other places you use safec headers) to use this format:
```
#ifdef USE_LIBSAFEC
#define restrict __restrict
#include <libsafec/safe_str_lib.h>
#elif USE_SLIBC
#include <slibc/string.h>
#else
#error "No safe C library defined, define either USE_LIBSAFEC or USE_SLIBC"
#endif
```